aemedge/blocks/accordion/accordion.css

```diff
@@ -1,57 +1,48 @@
-.accordion details {
-  border: 1px solid var(--dark-color);
+.accordion p {
+  margin: 0
 }
 
-/* stylelint-disable-next-line no-descending-specificity */
-.accordion details + details {
-  margin-top: 16px;
+.accordion details {
+  border-bottom: 1px solid var(--udexColorGrey3);
+  width: 100%
 }
 
 .accordion details summary {
   position: relative;
-  padding: 0 16px;
-  padding-right: 48px;
+  padding: var(--udexSpacer12) var(--udexSpacer12) var(--udexSpacer12) var(--udexSpacer8);
   cursor: pointer;
   list-style: none;
   overflow: auto;
-  transition: background-color 0.2s;
+  display: inline-flex;
+  align-items: center;
+  font-size: var(--udexTypographyHeadingMediumXXXSFontSize);
+  font-weight: var(--udexTypographyFontWeightMedium);
+  box-sizing: border-box;
+  width: 100%;
 }
 
-.accordion details[open] summary {
-  background-color: var(--light-color);
-}
+.accordion .accordion-arrow {
+  box-sizing: border-box;
+  display: inline-block;
 
-.accordion details summary:focus,
-.accordion details summary:hover {
-  background-color: var(--dark-color);
-}
+  /* Helps ensure icon and text are aligned properly */
+  line-height: var(--udexTypographyHeadingMediumXXXSFontSize);
+  width: 42px;
+  height: 42px;
 
-.accordion details summary::-webkit-details-marker {
-  display: none;
+  /* Icon image should be 18px x 18px */
+  padding: calc((42px - 18px) / 2);
+  margin-right: var(--udexSpacer12);
 }
 
-.accordion details summary::after {
-  content: "";
-  position: absolute;
-  top: 50%;
-  right: 18px;
-  transform: translateY(-50%) rotate(135deg);
-  width: 9px;
-  height: 9px;
-  border: 2px solid;
-  border-width: 2px 2px 0 0;
-  transition: transform 0.2s;
+.accordion details[open] .accordion-arrow {
+  transform: rotate(90deg);
 }
 
-.accordion details[open] summary::after {
-  transform: translateY(-50%) rotate(-45deg);
+.accordion details summary::-webkit-details-marker {
```

Comment:
Why require a vendor prefix pseudo element?

Reply:
This was part of the styling provided in the base implementation of accordion, I assume this does something to hide some webkit accordion browser element and didn't want to remove.

Reply:
Check if it does nothing then remove it. 
@mhaack It is discouraged to use vendor prefixes. Can we check why it is in reference code?

Reply:
Lets not complicate it, se https://pages.github.tools.sap/UDEx/digital-design-system/?path=/docs/patterns-actions-accordion--docs that also has no transition. Let's keep it simpler.

> It is discouraged to use vendor prefixes
If this is for some styles only I would be fine. I that impacts functionality we should not use it.

Reply:
> Lets not complicate it, se https://pages.github.tools.sap/UDEx/digital-design-system/?path=/docs/patterns-actions-accordion--docs that also has no transition. Let's keep it simpler.
> 
> > It is discouraged to use vendor prefixes
> > If this is for some styles only I would be fine. I that impacts functionality we should not use it.

Sure. On a side note I also see an "Expand All" button in their implementation. Should we also add that?
<img width="996" alt="Screenshot 2024-03-26 at 14 47 04" src="https://github.com/urfuwo/hlx-test/assets/41549648/af5cf2ec-be3e-4753-913b-4777d869fb57">

Reply:
I have removed the prefix as it nothing on Chrome or Firefox. Perhaps someone with a Mac can check on Safari and see if we should add it back, @saurabh-khare would you be able to confirm?

Reply:
> > Lets not complicate it, se https://pages.github.tools.sap/UDEx/digital-design-system/?path=/docs/patterns-actions-accordion--docs that also has no transition. Let's keep it simpler.
> > > It is discouraged to use vendor prefixes
> > > If this is for some styles only I would be fine. I that impacts functionality we should not use it.
> 
> Sure. On a side note I also see an "Expand All" button in their implementation. Should we also add that? <img alt="Screenshot 2024-03-26 at 14 47 04" width="996" src="https://private-user-images.githubusercontent.com/41549648/316886831-af5cf2ec-be3e-4753-913b-4777d869fb57.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0NjE0MjIsIm5iZiI6MTcxMTQ2MTEyMiwicGF0aCI6Ii80MTU0OTY0OC8zMTY4ODY4MzEtYWY1Y2YyZWMtYmUzZS00NzUzLTkxM2ItNDc3N2Q4NjlmYjU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI2VDEzNTIwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4N2M5ZDJhMWYyY2Y5ZDg1YTA1NjU2MGE2ODczMzU1YWZjNTM4NjI0MzA4ZjlkMWQyNmY3YTQ0MzhlYWEyOTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Qa9oEp9qnO2oaGHThbNipbSVXimzx0DbHQZOIrn5Cls">

There is a follow up ticket raised for this https://github.com/urfuwo/hlx-test/issues/385

Reply:
> I have removed the prefix as it did nothing on Chrome or Firefox. Perhaps someone with a Mac can check on Chrome/Firefox/Safari and see if we should add it back, @saurabh-khare would you be able to confirm?

Arrghh I see ugly arrows in Safari. Add it back. I'll see later how can we avoid it.
<img width="1069" alt="Screenshot 2024-03-26 at 15 08 31" src="https://github.com/urfuwo/hlx-test/assets/41549648/34a8cfd7-5597-491f-aa62-18b7afb15175">